curriculum/challenges/english/10-coding-interview-prep/rosetta-code/jortsort.md

```diff
@@ -8,7 +8,7 @@ dashedName: jortsort
 
 # --description--
 
-jortSort is a sorting toolset that makes the user do the work and guarantees efficiency because you don't have to sort ever again. It was originally presented by Jenn "Moneydollars" Schiffer at the prestigious [JSConf](https://www.youtube.com/watch?v=pj4U_W0OFoE).
+jortSort is a sorting toolset that makes the user do the work and guarantees efficiency because you don't have to sort ever again. It was originally presented by Jenn "Moneydollars" Schiffer at the prestigious JSConf.
```

Comment:
Maybe add a date when the Conference took place?

Reply:
I have added 2014 because the video is publish 9 of June, and I am not sure if it's published the same day it took place😓.